package.xml

```diff
@@ -7,7 +7,7 @@
   <maintainer email="gsaurel@laas.fr">Guilhem Saurel</maintainer>
   <author email="maximilien.naveau@toward.fr">Maximilien Naveau</author>
 
-  <license>BSD-2</license>
+  <license>BSD-3</license>
```

Comment:
BSD 2 and BSD 3 are not licenses. please take either BSD-2-Clause or BSD-3-Clause, ref. https://spdx.org/licenses/